contracts/core/RebalancingSetToken.sol

```diff
@@ -241,8 +256,36 @@ contract RebalancingSetToken is
         // Must be in Rebalance state to call settlement
         require(rebalanceState == State.Rebalance);
 
+        // Make sure all currentSets have been rebalanced
+        require(remainingCurrentSets == 0);
+
+        // Create ICore object
+        ICore core = ICore(ISetFactory(factory).core());
+
+        // Issue nextSet and deposit in Vault
+        uint256 issueAmount;
+        (issueAmount, unitShares) = calculateNextSetIssueQuantity();
+        core.issue(
+            nextSet,
+            issueAmount
+        );
+
+        // Ensure allowance to transfer sets to Vault
```

Comment:
Don't need comment.

Reply:
Don't see the reason to be removing comments. They're free in my mind.

Reply:
If the code is clear enough, we don't need explanatory comments (which is a good thing). I'm not opinionated about this one. So up to you.

Reply:
I think above you talked about maintaining consistency and a long time ago we agreed on comments on every line and not assuming how obvious it is.

That being said, the comments could have a bit more color. In the case above, it could be, "Creating pointer to Core to Issue next set and Deposit into vault". This one could be: "Ensure transfer proxy has enough spender allowance to move issued nextSet to vault"